app/onepassword_events_api/appserver/static/javascript/views/app.js

```diff
@@ -67,68 +58,49 @@ export default class SetupPage extends React.Component {
   validateJWT(token) {
     const tokenComponents = token.split(".");
     if (tokenComponents.length !== 3) {
-      return "Invalid JSON Web Token - too short";
+      return "This doesn't look like a valid JSON Web Token.";
     }
     let payload;
     try {
       payload = JSON.parse(atob(tokenComponents[1]));
     } catch (error) {
-      return "Invalid JSON Web Token - " + error.message;
+      return "This doesn't look like a valid JSON Web Token.";
     }
     if (!payload.aud || payload.aud.length !== 1) {
-      return "Invalid JSON Web Token - missing aud";
+      return "This doesn't look like a valid JSON Web Token.";
     }
     if (payload.aud[0] === audienceDEPRECATED) {
       return "Please generate a new token";
     }
   }
 
   render() {
-    return e("div", null, [
-      e(
-        "h1",
-        null,
-        "1Password Events API for Splunk Setup Page - Version 1.5.3"
-      ),
-      e("div", null, [
-        e("div", { className: "warning" }, [
-          e("h3", null, [
-            "Your other Splunk apps or add-ons may be able to access your Events API token. Make sure you trust them before you add your token.",
-          ]),
-        ]),
-      ]),
-      e("div", null, [
-        e("form", { onSubmit: this.handleSubmit }, [
-          e("label", null, [
-            e("h3", null, ["Events API Token"]),
-            e("input", {
-              type: "text",
-              value: this.state.authToken,
-              onChange: (e) => {
-                this.setState({
-                  authToken: e.target.value,
-                });
-              },
-            }),
-          ]),
-          e("input", { type: "submit", value: "Submit" }),
-        ]),
-      ]),
-      this.state.error && e("div", { className: "error" }, this.state.error),
-      this.state.success &&
-        e("div", { className: "success" }, [
-          "Your token has been successfully updated. If this is the first time you're setting up 1Password Events API for Splunk, you'll have to enable the scripted inputs. If 1Password Events API for Splunk has already been setup, you'll have to disable and re-enable the scripted inputs for the changes to take effect.",
-          e("br"),
-          e("br"),
-          "For more information, check out the support article ",
-          e(
-            "a",
-            {
-              href: "https://support.1password.com/events-reporting-splunk",
-            },
-            "here."
-          ),
-        ]),
-    ]);
+    return e(
+      Wizard,
```

Comment:
We should push all this down into the Wizard component, since it's already so tightly coupled to the setup flow (pulling the step `description`/`warning`/etc up to the parent is not really useful to configure there). In the future we can then decide to create a more reusable wizard that takes components for each step rather than prop configurations like we have here. Components makes it a lot easier to maintain as the steps start to diverge from one another.